WalletWasabi.Gui/Controls/WalletExplorer/CoinViewModel.cs

```diff
@@ -140,17 +137,38 @@ public string ToolTip
 			{
 				switch (Status)
 				{
-					case SmartCoinStatus.Confirmed: return "This coin is confirmed.";
-					case SmartCoinStatus.Unconfirmed: return "This coin is unconfirmed.";
-					case SmartCoinStatus.MixingOnWaitingList: return "This coin is waiting for its turn to be coinjoined.";
-					case SmartCoinStatus.MixingBanned: return $"The coordinator banned this coin from participation until {Model?.BannedUntilUtc?.ToString("yyyy - MM - dd HH: mm", CultureInfo.InvariantCulture)}.";
-					case SmartCoinStatus.MixingInputRegistration: return "This coin is registered for coinjoin.";
-					case SmartCoinStatus.MixingConnectionConfirmation: return "This coin is currently in Connection Confirmation phase.";
-					case SmartCoinStatus.MixingOutputRegistration: return "This coin is currently in Output Registration phase.";
-					case SmartCoinStatus.MixingSigning: return "This coin is currently in Signing phase.";
-					case SmartCoinStatus.SpentAccordingToBackend: return "According to the Backend, this coin is spent. Wallet state will be corrected after confirmation.";
-					case SmartCoinStatus.MixingWaitingForConfirmation: return "Coinjoining unconfirmed coins is not allowed, unless the coin is a coinjoin output itself.";
-					default: return "This is impossible.";
+					case SmartCoinStatus.Confirmed:
+						return "This coin is confirmed.";
+
+					case SmartCoinStatus.Unconfirmed:
+						return "This coin is unconfirmed.";
+
+					case SmartCoinStatus.MixingOnWaitingList:
+						return "This coin is waiting for its turn to be coinjoined.";
+
+					case SmartCoinStatus.MixingBanned:
+						return $"The coordinator banned this coin from participation until {Model?.BannedUntilUtc?.ToString("yyyy - MM - dd HH: mm", CultureInfo.InvariantCulture)}.";
+
+					case SmartCoinStatus.MixingInputRegistration:
+						return "This coin is registered for coinjoin.";
+
+					case SmartCoinStatus.MixingConnectionConfirmation:
+						return "This coin is currently in Connection Confirmation phase.";
+
+					case SmartCoinStatus.MixingOutputRegistration:
+						return "This coin is currently in Output Registration phase.";
+
+					case SmartCoinStatus.MixingSigning:
+						return "This coin is currently in Signing phase.";
+
+					case SmartCoinStatus.SpentAccordingToBackend:
+						return "According to the Backend, this coin is spent. Wallet state will be corrected after confirmation.";
+
+					case SmartCoinStatus.MixingWaitingForConfirmation:
+						return "Coinjoining unconfirmed coins is not allowed, unless the coin is a coinjoin output itself.";
+
+					default:
+						return "This is impossible.";
```

Comment:
The previous was much better readable.

Reply:
It was CodeMaid. I can revert it but the problem is whenever you save this file it will happen again.

Reply:
We could turn off this rule in CodeMaid: https://github.com/zkSNACKs/WalletWasabi/blob/master/CodeMaid.config

Reply:
Which rule?

Reply:
I didn't check. I assumed if it's doing this, then it must be configurable.